app/services/liquid_assigns_service.rb

```diff
@@ -0,0 +1,24 @@
+class LiquidAssignsService
+  attr_accessor :project
+
+  def initialize(project)
+    @project = project
+  end
+
+  def assigns
+    result = {}
+    result['project'] = ProjectDrop.new(project)
+    result['issues'] = project.issues.map { |issue| IssueDrop.new(issue) }
+    result['nodes'] = project.nodes.user_nodes.map { |node| NodeDrop.new(node) }
+    result['evidences'] = project.evidence.map { |evidence| EvidenceDrop.new(evidence) }
+    result['tags'] = project.tags.map { |tag| TagDrop.new(tag) }
+
+    if defined?(Dradis::Pro)
```

Comment:
What about passing in kwargs instead? 
Ex.
`@team = args.fetch(:team, nil)` in the initializer, and
`result['team'] = TeamDrop.new(project.team) if team` 
on L19

Reply:
I like that better than the `Dradis::Pro` check but listing all the args sort of defeats the convenience this brings for the caller. If a caller has to instantiate the class with the following:
```
LiquidAssignsService.new(
  project: project,
  team: project.team,
  document_properties: project.content_library.properties
)
```

Then it starts to look like:
```
result['project'] = ...
result['document_properties'] = ...
result['team'] = ...
```

just without the drop?

Reply:
I'd extract this to a `#assigns_pro` that only gets called if `defined?` as we do in other places.